about/process/branch-naming.rst

```diff
@@ -0,0 +1,51 @@
+Branch-naming convention
+========================
+
+Every repo's readme should state which branch-naming-convention is used and possible deviations from the norm.
+
+Click-Packages
+--------------
+
+Software that is exclusively distributed as a click-package (and not also as a deb) only uses one ``master`` branch that is protected. Separate temporary development branches with arbitrary descriptive names can be created and merged into master when the time comes. For marking and archiving milestones in development history, ideally Git tags or GitHub releases should be used.
+
+Deb-Packages
+------------
+
+To make most efficient use of our CI system, a special naming convention for git-branches is used.
+
+For pre-installed Ubuntu Touch components, deb-packages are used whereever possible. This includes coreapps, since they can still be independently updated using click-package downloads from the openstore. This policy allows us to make use of the powerful debian build system to resolve dependencies.
```

Comment:
whereever should be wherever

Debian is a proper noun and should be capitalized

Do we want to say "coreapps" instead of core apps?